docs/example.ts

```diff
@@ -63,20 +101,27 @@ async function setup(): Promise<{
   const rawCtype: ICType = {
     schema: ctypeSchema,
     hash: ctypeHash,
-    owner: attester.address,
+    owner: attesterOnChainDid.did,
   }
 
   // Build the CType object
   const ctype = new Kilt.CType(rawCtype)
-
   // Store ctype on blockchain
   // signAndSubmitTx can be passed SubscriptionPromise.Options, to control resolve and reject criteria, set tip value, or activate re-sign-re-send capabilities.
   // ! This costs tokens !
   // Also note, that the completely same ctype can only be stored once on the blockchain.
```

Comment:
I would rephrase this to just say "the same CType". No need to specify completely, in my opinion.

Reply:
Agreed.